bin/node/src/commands/bundled.rs

```diff
@@ -28,6 +30,9 @@ pub enum BundledCommand {
         // Directory to write the account data to.
         #[arg(long, value_name = "DIR")]
         accounts_directory: PathBuf,
+        /// Constructs the genesis block from the given toml file.
+        #[arg(long, env = ENV_GENESIS_CONFIG_FILE, value_name = "FILE")]
+        genesis_config_file: Option<PathBuf>,
```

Comment:
Is the PR name erroneous because it says _**test**_ genesis configuration toml file? As in this flag could/would be used to set up any network including our devnet if we re-genesis it for example.